src/plugins/europeana/record.js

```diff
@@ -20,15 +20,20 @@ export const BASE_URL = process.env.EUROPEANA_RECORD_API_URL || 'https://api.eur
  * @return {Object[]} Key value pairs of the metadata fields.
  */
 function coreFields(proxyData, providerAggregation, entities) {
-  return Object.freeze(lookupEntities(omitBy({
-    edmDataProvider: { url: providerAggregation.edmIsShownAt, value: providerAggregation.edmDataProvider },
+  const fields = lookupEntities(omitBy({
+    edmDataProvider: providerAggregation.edmDataProvider,
     dcContributor: proxyData.dcContributor,
     dcCreator: proxyData.dcCreator,
     dcPublisher: proxyData.dcPublisher,
     dcSubject: proxyData.dcSubject,
     dcType: proxyData.dcType,
     dctermsMedium: proxyData.dctermsMedium
-  }, isUndefined), entities));
+  }, isUndefined), entities);
+
+  fields.edmDataProvider = {
+    url: providerAggregation.edmIsShownAt, value: fields.edmDataProvider
+  };
```

Comment:
Wouldn't this always override the `edmDataProvider` field above?

Reply:
It does the entity lookup first, above, then preserves the result of that here in `value`, adding isShownAt to `url`.